tests/unit/kvmagent/test_localstorage_handlers.py

```
@@ -0,0 +1,1189 @@
+_ = sys.modules.setdefault("traceable_shell", MagicMock())
+_ = sys.modules.setdefault("report", MagicMock())
+_ = sys.modules.setdefault("linux", MagicMock())
+_ = sys.modules.setdefault("bash", MagicMock())
+_ = sys.modules.setdefault("shell", MagicMock())
+
+try:
+    localstorage = cast(
+        _LocalStorageModule,
+        cast(object, importlib.import_module("kvmagent.plugins.localstorage")),
+    )
+except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)